core/shared/platform/windows/platform_internal.h

```diff
@@ -89,18 +89,15 @@ os_mem_decommit(void *ptr, size_t size);
 #define strncasecmp _strnicmp
 #define strcasecmp _stricmp
 
+#if WASM_DISABLE_BLOCK_INSN_INTERRUPT == 0
+#define OS_ENABLE_BLOCK_INSN_INTERRUPT
+#endif
```

Comment:
Windows doesn't support posix like API, had better not change this file currently

Reply:
isn't `setjmp` supported on Windows? https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/setjmp

Reply:
Windows signal is a little more complex, we tried using setjmp/longjmp before, but found sometimes longjmp doesn't work, it cannot jump the place set by setjmp.
Refer to https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/signal?view=msvc-160
It mentions:
`Don't use longjmp unless the interrupt is caused by a floating-point exception (that is, sig is SIGFPE). In this case, first reinitialize the floating-point package by using a call to _fpreset.`

My suggestion is to enable posix platforms first, and enable windows in the future.

Reply:
right, it's also reported in the `longjmp` https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/longjmp